cli/commands/cluster/member-list.go

```diff
@@ -33,7 +33,7 @@ func MemberListCommand(cli *cli.SensuCli) *cobra.Command {
 	return cmd
 }
 
-func printToTable(result interface{}, w io.Writer) {
+func printMemberListToTable(result interface{}, w io.Writer) {
 	memberList := result.(*clientv3.MemberListResponse)
```

Comment:
Should we check for an error on this type assertion? `memberList, ok := result.(*clientv3.MemberListResponse)`